core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -457,6 +457,7 @@ export class OptimizedSsrEngine {
           provide: EXPRESS_SERVER_LOGGER,
           useValue: this.logger,
         },
+        ...options?.providers,
```

Comment:
Now will it break, if customer doesn't pass any custom providers? Just asking, because I saw the changes in the unit tests...
```suggestion
        ...options?.providers ?? [],
```

Reply:
good catch